fido/fido.py

```diff
@@ -166,14 +174,18 @@ def _escape_char(self,c):
             (high, low) = divmod(ord(c), 16)
             return '\\x' + self._hex[high] + self._hex[low]
 
-    def escape(self,string):
-        "Escape characters in pattern that are non-printable, non-ascii, or special for regexes."
+    def escape(self, string):
+        """
+        Escape characters in pattern that are non-printable, non-ascii, or
+        special for regexes.
```

Comment:
Nitpick: This should probably be one line, since the first line of a docstring is often treated as a summary by tools and breaking the sentence up will look odd.